foursight_core/environment.py

```diff
@@ -44,7 +44,8 @@ def get_environment_and_bucket_info(self, env_name, stage):
             }
             return env_entry
         else:
-            raise Exception("malformatted environment info on s3 for key %s" % env_name)
+            raise Exception(f'malformatted environment info on s3 for key {env_name}\n'
```

Comment:
I usually prefer sentence case and punctuation for errors, so uppercase initial and a colon after `{env_name}`. But also, does this really need a continuation line? I guess maybe it's there to emphasize the `\n`?